packages/vx-responsive/src/enhancers/withParentSize.js

```diff
@@ -28,7 +28,7 @@ export default function withParentSize(BaseComponent) {
 
     resize(event) {
       if (this.container) {
-        var boundingRect = this.container.getBoundingClientRect();
+        const boundingRect = this.container.getBoundingClientRect();
```

Comment:
this might be another instance where you need a `var` so it can be used in the closure